Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                        {
+                            ag.ExternalID2 = tsTest.ID;//the test case instance ID in the test set- used for exporting the execution details
+                        }
+                    }
+                    else
+                    {
+                        foreach (ActivityIdentifiers actIdent in ag.ActivitiesIdentifiers)
+                        {
+                            ExportActivityAsTestStep(ImportFromQC.GetQCTest(ag.ExternalID),(Activity) actIdent.IdentifiedActivity);
+                        }
+                    }
+                }
+                return true;
+            }
+            catch (Exception ex)
+            {
+                result = "Unexpected error occurred- " + ex.Message;
+                Reporter.ToLog(eLogLevel.ERROR, "Failed to export the " + GingerDicser.GetTermResValue(eTermResKey.BusinessFlow) + " to QC/ALM", ex);
+                return false;
+            }
+        }
+        
+
+
+        #region swapnas code
+        
+        // Create New Defect in Defects Tab QC 
+        public static int CreateDefect(String Status, String Summary, String DetectedBy, String Version)
+        {
+            try
+            {
+                BugFactory bugFactory = mTDConn.BugFactory;
+
+                //start of get bug mandatory fields
+                Customization field = (Customization)mTDConn.Customization;
+                CustomizationFields f = (CustomizationFields)field.Fields;
+
+                List BugFieldsList = f.get_Fields("BUG");
+                Dictionary<string, string> MandatoryFields = new Dictionary<string, string>();
+                Dictionary<string, List<string>> MandatoryListSelections = new Dictionary<string, List<string>>();
+                foreach (CustomizationField BugField in BugFieldsList)
+                {
+                    if (BugField.IsRequired)
+                    {
+                        List<string> FieldList = new List<string>();
+                        if (BugField.List != null && BugField.List.RootNode.Children.Count > 0)
+                        {
+                            CustomizationListNode lnode = BugField.List.RootNode;
+                            List cNodes = lnode.Children;
+                            foreach (CustomizationListNode ccNode in cNodes)
+                            {
+                                //adds list of valid selections of Field
+                                FieldList.Add(ccNode.Name);
+                            }
+                        }
+                        if (!MandatoryFields.ContainsKey(BugField.UserLabel))
+                        {
+                            MandatoryFields.Add(BugField.UserLabel, BugField.ColumnName);
+                            if (FieldList.Count > 0)
+                                MandatoryListSelections.Add(BugField.UserLabel, FieldList);
+                            else MandatoryListSelections.Add(BugField.UserLabel, null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'else' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)